src/app/app.module.ts

```diff
@@ -18,6 +18,7 @@ import { initializeAuth } from './shared/auth/init-auth';
     SharedModule,
     DashboardModule,
     KeycloakAngularModule,
+    FormsModule,
```

Comment:
Can you move this to only be imported in the SharedModule?

Reply:
Removed it from here.